### ISSUE TITLE
code preview

### ISSUE THREAD

**Issue by afrieirham** [origin-space/originui#5]
When I copy the code, it includes the imports and the creation of the components. 
Sometimes I just want to copy the JSX, having a section where I can view the code would be a great addition to this.

Something like this 👇

https://github.com/user-attachments/assets/9d5067a4-a85a-423f-b33d-a0e3af35fd83

ps - i'm willing to do the implementation if you are okay with the suggestion.

**Comment by davidepacilio** [comment]
Great suggestion, @afrieirham! Thanks so much for sharing it with us. We’re still in the exploration phase, so things may shift around a bit as we go, but if you’d like to show us a preview of the implementation, we’d love to check it out.

**Comment by baginski11** [comment]
How about a dropdown menu with a selection of:
- Copy Import
- Copy Component Code

**Comment by downright-development** [comment]
I would just recommend following the standards of ShadCN when it lays out components & code.

**Comment by Hat52** [comment]
I agree with downright, I think instead of creating two different section for code we should stick with the only one.

**Comment by pasqualevitiello** [comment]
Closing the issue. Feature was added with https://github.com/origin-space/originui/commit/c2617fe1ff57c96303e7acd8e77921387ea81267
